lib/path.h

```diff
@@ -118,6 +118,13 @@ class CPPCHECKLIB Path {
       */
     static std::string getRelativePath(const std::string& absolutePath, const std::vector<std::string>& basePaths);
 
+    /**
+      * @brief Checks if a Folder exists
+      * @param path Path to be checked if it is a folder
+      * @return true if given path is a folder
+      */
+    static bool folderExist(const std::string &path);
```

Comment:
there is a `fileExists` at the bottom so I think it's logical to put this above/below that.

Reply:
I found similar functions in FileLister and moved them to Path adapted test etc.